index.bs

```diff
@@ -672,7 +672,7 @@ Unlike WebGPU, this API does not intrinsically support custom shader authoring;
 
 The WebGPU API identifies <a href="https://gpuweb.github.io/gpuweb/#privacy-machine-artifacts">machine-specific artifacts</a> as a privacy consideration. Similarly, the WebNN API's compute unit scheduling may under certain circumstances introduce a fingerprint. However, similarly to WebGPU, such fingerprints are identical across most or all of the devices of each vendor, mitigating the concern. Furthermore, software implementations can be used to further eliminate such artifacts.
 
-The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. [=Device type=] normatively indicates the kind of device and is either {{MLDeviceType/"cpu"}} or {{MLDeviceType/"gpu"}}. If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown, thus this type can in some cases add two bits of entropy to the fingerprint. [=Power preference=] indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
+The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. [=Device type=] normatively indicates the kind of device is one of: {{MLDeviceType/"cpu"}}, {{MLDeviceType/"gpu"}}, {{MLDeviceType/"npu"}}. If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown, and thus this type can in some cases add two bits of entropy to the fingerprint. [=Power preference=] indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
```

Comment:
@anssiko also proposed adding:

""ISSUE: MLContextOptions is under active development and the design is expected to change informed by further implementation experience and new use cases from the wider web community. The Working Group is considering additional API surface to allow definition of a fallback device, multiple devices in a preferred order, or an exclusion of a specific device. Other considerations under discussion include error handling, ultimate
fallback and quantized operators. Feedback is welcome on any of these design considerations from web developers, library authors, OS and hardware vendors, and other stakeholders via GitHub issue [#623](https://github.com/webmachinelearning/webnn/issues/623)."

Reply:
Here's the Bikeshed syntax to create such an in-line issue block:
```
Issue(623): MLContextOptions is under active development and the design is expected to change informed by further implementation experience and new use cases from the wider web community. The Working Group is considering additional API surface to allow definition of a fallback device, multiple devices in a preferred order, or an exclusion of a specific device. Other considerations under discussion include error handling, ultimate
fallback and quantized operators. Feedback is welcome on any of these design considerations from web developers, library authors, OS and hardware vendors, and other stakeholders via GitHub:
```
Bikeshed magic appends the following link at the end of the block (thus trailing ":"):
[[Issue #623]](https://github.com/webmachinelearning/webnn/issues/623)

If we want more control over formatting or where the link is placed, we can use a plain old `<a>`.

I'd propose to place this block as the first thing in this section: https://www.w3.org/TR/webnn/#api-mlcontextoptions

Reply:
Curious, I don't see the clickable pipeline result at the bottom of these code reviews anymore. Now I have to:
- click "Show all checks"
- click "Details"
- click "Summary"
- download `spec-prod-result.zip` from the artifacts
- extract `webnn.w3c/index.html`
- double click the .html file

So from 1-click step to 6 steps. Anyway, the issue text looks right.